source/voice_control/thirteen-usd-voice-remote.markdown

```diff
@@ -6,18 +6,35 @@ This tutorial will guide you to turn an ATOM Echo into the
 world's most private voice assistant. Pick up the tiny device to talk to
 your smart home. Issue commands and get responses!
 
-<lite-youtube videoid="w6QxGdxVMJs" videotitle="$13 voice remote for Home Assistant
-"></lite-youtube>
-
 ## Required material
 
-- Home Assistant 2023.5 or later
+- Home Assistant 2023.10
 - [Home Assistant Cloud](https://www.nabucasa.com) or a manually configured [Assist Pipeline](/voice_control/voice_remote_local_assistant)
 - The password to your 2.4&nbsp;GHz Wi-Fi network
 - Chrome (or a Chromium-based browser like Edge) on desktop (not Android/iOS) 
 - [M5Stack ATOM Echo Development Kit](https://shop.m5stack.com/products/atom-echo-smart-speaker-dev-kit?ref=NabuCasa)
 - USB-C cable to connect the ATOM Echo
 
+## Installing the openWakeWord add-on
+
+As a first step, you need to install the openWakeWord add on. This must be installed before setting up the ATOM Echo.
+
+1. Go to {% my supervisor_addon addon="openwakeword" title="**Settings** > **Add-ons** > **openWakeWord**" %} and select **Install**.
+2. Start the add-on.
+3. Go to {% my integrations title="**Settings** > **Devices & Services**" %}.
+   - Under **Discovered**, you should now see the **openWakeWord** integration.
+   - Select **Configure** and **Submit**.
+4. To define the wake word enging, under **Wake word**, select **openwakeword**. 
```

Comment:
Should we mention this is in the Voice Assistants settings within a pipeline?

Reply:
oh, yes. something went missing here. Thank you for the catch.